packages/zcli-apps/src/commands/apps/create.ts

```diff
@@ -53,7 +53,8 @@ export default class Create extends Command {
         const configParams = allConfigs?.parameters || {} // if there are no parameters in the config, just attach an empty object
 
         const settings = manifest.parameters ? await getAppSettings(manifest, configParams) : {}
-        const installed = await request.requestAPI('api/v2/apps/installations.json', {
+        const product = Object.keys(manifest.location)[0]
```

Comment:
Wondering if we need to consider multiple locations use-case here? ie 
```
  "location": {
    "support": {
      "ticket_editor": "assets/iframe.html",
      "nav_bar": "assets/iframe.html"
    },
    "sell": {
      "top_bar": "assets/iframe.html"
    }
  },
```

Reply:
A good point - although I'm not sure that zcli supported multiple locations prior to this change.  I can certainly look into this though, as this is the sort of feature that we probably should have.